ted_sws/notice_validator/services/xpath_coverage_runner.py

```diff
@@ -0,0 +1,101 @@
+import re
+import tempfile
+import xml.etree.ElementTree as ET
```

Comment:
Where you use ET ? But for future use another alias like ElementTree more meaningful than ET.

Reply:
Was for initial version of xpaths retrieval. Since there is already implemented one, I've adapted (ET is just garbage). Will remove it and make a commit